notebook/log.py

```diff
@@ -46,5 +50,11 @@ def log_request(handler):
     if status >= 500 and status != 502:
         # log all headers if it caused an error
         log_method(json.dumps(dict(request.headers), indent=2))
-    log_method(msg.format(**ns))
+    if _enable_json_logs:
```

Comment:
Just because this is true doesn't mean that json-logging is installed. If it's not, the request log line ends up looking like this:

```
[W 17:04:57.097 NotebookApp] {'status': 404, 'method': 'GET', 'ip': '127.0.0.1', 'uri': '/nbextensions/widgets/notebook/js/extension.js', 'request_time': 6.743431091308594, 'referer': 'http://localhost:8888/notebooks/Untitled.ipynb?kernel_name=python3'}
```

Also note that with the json logging we're not formatting the request time to 2 decimal places:

`'request_time': 6.743431091308594`